### DISCUSSION TITLE
Welcome to M480BSP Discussions!

### DISCUSSION THREAD

**Discussion by yachen** [OpenNuvoton/M480BSP#24] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by alexcwrl** [comment]
Hi everybody. 
Hi Yachen.

I am a software and hardware designer for Li-ion batteries and motor controllers. No I am designing a balancer (BMS) board for 8s battery. It is the start of a whole BMS boards' family, combined with motor controllers, and separate.

I have problem (yes!))) to start my own "HSUSBD_VCOM_SerialEmulator", based on your sample. My device is M483SIDAE.
I tried to copy source files and built them into my project. I tried to run the sample code directly. Result is same - "unknown USB device acts in wrong way" or something like that.
I thought may be my hardware is malfunction - but "HSUSBD_Audio10_Headset" runs fine from the point of view of USB device.
